domains/mc.notanicepenguin.json

```
@@ -0,0 +1,14 @@
+{
+  "owner": {
+    "github": "notanicepenguin"
```

```suggestion
    "username": "notanicepenguin"
```
Username field is required.